ext/ftp/php_ftp.c

```diff
@@ -99,15 +99,13 @@ static void ftp_object_destroy(zend_object *zobj) {
 
 PHP_MINIT_FUNCTION(ftp)
 {
-#ifdef HAVE_FTP_SSL
-#if OPENSSL_VERSION_NUMBER < 0x10101000 && !defined(LIBRESSL_VERSION_NUMBER)
+#if defined(HAVE_FTP_SSL) && !defined(LIBRESSL_VERSION_NUMBER)
 	SSL_library_init();
```

Comment:
These functions inside this #if block are either deprecated or do nothing anymore on OpenSSL versions >= 1.1.0.
So I think this should've been removed, i.e. `OPENSSL_VERSION_NUMBER < 0x10101000` is always false now.

Reply:
FTR: https://github.com/php/php-src/pull/13793

Reply:
This PR does not contain LibreSSL bump: See https://github.com/php/php-src/pull/13498#discussion_r1502634203

Reply:
Ah sorry this part could actually be removed - it's not defined - missed that.

Reply:
just went through it again and looks really like the only incorrect thing. Next step will be to bump LibreSSL and drop the code that is not needed.